snippets/code_to_review2.py

```diff
@@ -0,0 +1,73 @@
+import os
+from abc import abstractmethod
+import numpy as np
+
+class Analysis(object):
+    '''
```

Comment:
Don't know if this inheritance from `object` here is intentional? 
That's Py2 legacy not needed - and IMHO subject to inspection from code reviews.

Reply:
not intended and might be confusing, indeed, we should leave py2 behind, I'll remove it